bennettbot/job_configs.py

```diff
@@ -314,6 +319,12 @@
             },
         },
         "slack": [
+            {
+                "command": "emoji",
```

Comment:
nit: I would make the command "key"; `workflows key` is more intuitive (to me at least) than `workflows emoji`

Reply:
I was debating between the two for a while - will go with `workflows key` then!